YoutubePlaylistDownloader/GlobalUsings.cs

```diff
@@ -55,5 +55,4 @@
 global using FlowDirection = System.Windows.FlowDirection;
 global using HorizontalAlignment = System.Windows.HorizontalAlignment;
 global using Cursors = System.Windows.Input.Cursors;
-global using Brush = System.Windows.Media.Brush;
```

Comment:
Does it work correctly without this? Because I remember there was another class called Brush it always complained about

Reply:
It's because of habit I did a _Remove and Sort Usings_, but it still builds and runs without any apparent errors.